templates/security/compliance-automation.html

```diff
@@ -10,304 +14,539 @@
         </p>
-        <a class="p-button" href="https://wiki.ubuntu.com/Security/Features">See our security features</a>
-      </div>
-    </div>
+        <div class="p-section--shallow">
+          <div class="p-cta-block">
+            <a href="https://wiki.ubuntu.com/Security/Features">See our security features&nbsp;&rsaquo;</a>
+          </div>
+        </div>
+      {%- endif -%}
+
+    {%- endcall -%}
   </section>
 
   <section class="p-section">
-    <div class="row p-strip p-divider">
-      <h2>Available on-prem and in the cloud</h2>
-      <div class="col-4 col-medium-2">
-        <img src="https://assets.ubuntu.com/v1/9faa9b91-aws-logo.png"
-             alt="AWS"
-             style="width:auto;
-                    height: 6.5rem"
-             loading="lazy" />
-        <p>
-          <a href="/aws">Learn more about Ubuntu on AWS&nbsp;></a>
-        </p>
+    <div class="p-section--shallow">
+      <div class="u-fixed-width">
+        <hr class="p-rule" />
+        <h2>Available on-prem and in the cloud</h2>
       </div>
-      <div class="col-4 col-medium-2 p-divider__block">
-        <img src="https://assets.ubuntu.com/v1/21c19804-microsoft-azure-logo.png"
-             alt="Microsoft Azure"
-             style="width:auto;
-                    height: 6.5rem"
-             loading="lazy" />
-        <p>
-          <a href="/azure">Learn more about Ubuntu on Azure&nbsp;></a>
-        </p>
+    </div>
+    <div class="row">
+      <div class="col-start-large-4 col-3 col-medium-2">
+        <a href="/aws">
+          <div class="p-image-container is-highlighted u-hide--medium">
+            {{ image(url="https://assets.ubuntu.com/v1/7b612d59-aws.png",
+                      alt="",
+                      width="852",
+                      height="481",
+                      hi_def=True,
+                      loading="lazy") | safe
+            }}
+          </div>
+          <p>
+            Learn more about Ubuntu<br class="u-hide--small"/>on AWS&nbsp;&rsaquo;
+          </p>
+        </a>
       </div>
-      <div class="col-4 col-medium-2 p-divider__block">
-        <img src="https://assets.ubuntu.com/v1/92c62d40-lockup_GoogleCloud_FullColor_rgb_544x96px.png"
-             alt="Google Cloud"
-             style="width:auto;
-                    height: 2rem;
-                    margin-top: 2rem;
-                    margin-bottom: 2.5rem"
-             loading="lazy" />
-        <p>
-          <a href="/gcp">Learn more about Ubuntu on Google Cloud Platform&nbsp;></a>
-        </p>
+      <div class="col-3 col-medium-2">
+        <a href="/azure">
+          <div class="p-image-container is-highlighted u-hide--medium">
```

Comment:
This seems to satisfy Mattea's [comment](https://github.com/canonical/ubuntu.com/pull/14767#discussion_r1967267546):
![image](https://github.com/user-attachments/assets/9be22d78-1fca-47ce-a7d6-8071232b0671)

```suggestion
          <div class="p-image-container is-highlighted">
```